Sources/Clappr_iOS/Classes/Base/Layers/LayersCompositor.swift

```diff
@@ -0,0 +1,15 @@
+import UIKit
+
+class LayersCompositor {
+    private weak var rootView: UIView?
+    private var backgroundLayer: BackgroundLayer?
+    
+    func compose(inside rootView: UIView) {
+        self.rootView = rootView
+        
+        let backgroundLayer = BackgroundLayer()
+        self.backgroundLayer = backgroundLayer
```

Comment:
I think this `let backgroundLayer` doesn't need to be instantiated and the fact to create an instance of `BackgroundLayer` inside the method could show this method have more than one responsibility. But removing this, we have to refactor the `backgroundLayer` and the class. What do you think about it?

Reply:
What is your suggestion? To initialize `backgroundLayer` upon declaration to get rid of the optional value? If so, I do like this idea better.

Reply:
Yeap
=)

Reply:
Done in 3b71f1ef5d439bcfe14b4aebbe1a4721159d669e